publish.sh

```diff
@@ -19,24 +19,25 @@
 set -e
 
 echo "Prior to running this make sure that the NuttX repo is checked out"
-echo "to nuttx/master and the documentation html has been generated."
+echo "to nuttx and the documentation html has been generated."
 
 gem install bundler:2.1.2
 bundle config set path 'vendor/bundle'
 bundle install 
 bundle exec jekyll clean --source .
 bundle exec jekyll build --source .
 
-mkdir -p target/docs/
-cp -r nuttx/master/Documentation/_build/html target/docs/latest
+rm -rf target/docs
+mv docs target/
 
 COMMIT_HASH_WEB=`git rev-parse HEAD`
-COMMIT_HASH_NUTTX=`git -C nuttx/master rev-parse HEAD`
+COMMIT_HASH_NUTTX=`git -C nuttx rev-parse HEAD`
```

Comment:
This was being used for generating a commit message for the static site.  Do you think it would make sense to change this now to include all revs we are using in the message. So each iteration of the NuttX docs we would write to a file and then use that as a commit message.  Something like this:
```
website : $COMMIT_HASH_WEB
docs:
  * master: $COMMIT_NUTTX_{master}
  * nuttx-10.0.0-RC0:  $COMMIT_NUTTX_{nuttx-10.0.0-rc0}
```

Reply:
But is there any purpose for that message?

Reply:
Maybe it could be more useful to change the footer of the docs so that the build date and nuttx/ commit is shown on every page. What do you think?

Reply:
We need something.  I found it useful to keep track of what source was used to generate the release.  I can also put something together in a follow on PR.